packages/p2p/src/components/orders/orders.jsx

```diff
@@ -8,27 +8,42 @@ import './orders.scss';
 
 const Orders = ({ orders, params }) => {
     const [order_details, setDetails] = React.useState(null);
-    const showDetails = setDetails;
+    const [order_id, setOrderId] = React.useState('');
     const hideDetails = () => setDetails(null);
 
+    const setQueryDetails = input_order => {
+        // Changing query params
+        const new_url = `${window.location.protocol}//${window.location.host}${window.location.pathname}?order=${input_order.id}${window.location.hash}`;
```

Comment:
Same thing here, let's not do this, an in-memory router for this application would work better. Also, in any case this `new_url` would need to have the previous/existing params (`window.location.search`) appended as well